packages/js/product-editor/changelog/55621-fix-product-editor-ts-errors-4

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: dev
+
+`@woocommerce/data`: Enable TS check.
```

Comment:
We should update this changelog to reflect the changes within the `product-editor` package.
Given they are quite small setting it to tweak and just leaving a comment would make sense to.

Reply:
Good point! I updated it with dedb4101f8d0aedd89dacdcd505bf5e4ddc7bd95